Project.toml

```diff
@@ -59,4 +59,4 @@ StatsBase = "0.32, 0.33"
 StatsFuns = "0.8, 0.9"
 Tracker = "0.2.3"
 ZygoteRules = "0.2"
-julia = "1.3"
+julia = "1.3, 1.4, 1.5, 1.6"
```

Comment:
```suggestion
julia = "1.6"
```

Reply:
We don't have to be more restrictive than necessary 🤷‍♂️ We can easily drop support for < 1.6 when it requires special effort to support older versions. As long as it just works, I don't think we have to remove them.